ml/build.gradle

```diff
@@ -21,12 +21,11 @@ dependencies {
 
 
     compile "ai.h2o:mojo2-runtime-api:${mojoPipelineVersion}"
+    runtime "ai.h2o:mojo2-runtime-impl:${mojoPipelineVersion}"
```

Comment:
I was asked to test it. There is a breaking conflict between versions of the `com.google.protobuf` library. `hadoop-commons` (hadoop 2.7) utilizes version 2.5 and the new MOJO utilizes the version 3.7. It worked for me to make the dependency as `compile` and include and relocate the `com.google.protobuf` within the assembly jar.

Reply:
Can you share reproducible code please? It works on:

```
from pysparkling.ml import *
path = "/Users/kuba/devel/repos/sparkling-water/ml/src/test/resources/mojo2data/pipeline.mojo"
settings = H2OMOJOSettings(namedMojoOutputColumns=False)
mojo = H2OMOJOPipelineModel.createFromMojo(path, settings)

data = spark.read.csv("/Users/kuba/devel/repos/sparkling-water/examples/smalldata/prostate/prostate.csv", header=True)
mojo.transform(data).collect()
```

Reply:
Is this `pipeline.mojo` file produced by the latest version of DAI? I will send you the script that I tested on ;-)

Reply:
I see! This pipeline is using the toml config file -> I guess this error occurs for proto based pipelines :) Will add test for it and apply your suggestion, thanks!

Reply:
note: protobuf will be shadowed in mojo library, but it won't solve the issue with exceeding limit for pypi upload as we still bundle the libraries into our fat jar